website/pages/blog/announcing-cloudquery-aws-event-based-sync-beta.mdx

```diff
@@ -0,0 +1,125 @@
+
+![Configuring CloudQuery AWS Plugin for event-based sync](/images/blog/announcing-aws-event-based-sync/diagram.png)
+*Configuring CloudQuery AWS Plugin for event-based sync*
+
+With this setup, you get the fresh data within a few seconds of it becoming available in CloudTrail. 
+
+Here are the services and events supported at the moment:
+
+| Service | Event |
+| -- | -- |
+| ec2.amazonaws.com |	AttachInternetGateway |
+| ec2.amazonaws.com | AuthorizeSecurityGroupEgress |
+| ec2.amazonaws.com | AuthorizeSecurityGroupIngress |
+| ec2.amazonaws.com | CreateInternetGateway |
+| ec2.amazonaws.com | CreateSecurityGroup |
+| ec2.amazonaws.com | CreateSubnet |
+| ec2.amazonaws.com | CreateTags |
+| ec2.amazonaws.com | CreateVpc |
+| ec2.amazonaws.com | DeleteTags |
+| ec2.amazonaws.com | DetachInternetGateway |
+| ec2.amazonaws.com | ModifySubnetAttribute |
+| ec2.amazonaws.com | RevokeSecurityGroupEgress |
+| ec2.amazonaws.com | RevokeSecurityGroupIngress |
+| ec2.amazonaws.com | RunInstances |
+| iam.amazonaws.com | CreateGroup |
+| iam.amazonaws.com | CreateRole |
+| iam.amazonaws.com | CreateUser |
+| rds.amazonaws.com | CreateDBCluster |
+| rds.amazonaws.com | CreateDBInstance |
+
+## Getting Started
+
+This feature is available in closed beta at the moment, [sign up for early access](https://cloudquery.typeform.com/to/Dvdn6P2u).
+
+1. Configure a AWS CloudTrail Trail to send management events to a Kinesis Data Stream via CloudWatch Logs. The most straight forward way to do this is to use the CloudQuery provided CloudFormation template.
+
+```bash
+aws cloudformation deploy --template-file ./streaming-deployment.yml --stack-name <STACK-NAME> --capabilities CAPABILITY_IAM --disable-rollback --region <DESIRED-REGION>
+```
+
+2. Copy the ARN of the Kinesis stream. If you used the CloudFormation template you can run the following command:
+
+```bash
+aws cloudformation describe-stacks --stack-name <STACK-NAME> --query "Stacks[].Outputs" --region <DESIRED-REGION>
+```
+
+3. Define a `config.yml` file like the one below
+
+```yaml
+kind: source
+spec:  
+	name: "aws-event-based"  
+	registry: "local"  
+	path: <PATH/TO/BINARY>
+	tables:
+		- aws_ec2_instances
+		- aws_ec2_internet_gateways
+		- aws_ec2_security_groups
+		- aws_ec2_subnets
+		- aws_ec2_vpcs
+		- aws_ecs_cluster_tasks
+		- aws_iam_groups
+		- aws_iam_roles
+		- aws_iam_users
+		- aws_rds_instances
+destinations: ["postgresql"]  
+	skip_tables:    
+		- aws_iam_group_last_accessed_details    
+		- aws_iam_role_last_accessed_details    
+		- aws_iam_user_last_accessed_details  
+	spec:    
+		event_based_sync:      
+			- account:          
+					local_profile: "<ROLE-NAME>"
+				kinesis_stream_arn: "<OUTPUT-FROM-CLOUDFORMATION-STACK>"
+```
+
+4. Sync the data!
+
+```bash
+cloudquery sync config.yml
+```
+
+This will start a long lived process that will only stop when there is an error or you stop the process
+
+## Deploying in production
```

Comment:
I feel like we could move this section to the documentation for the feature, rather than the blog post, WDYT?

Reply:
I think this is quite important to manage the expectations. It doesn't really hurt, does it?

@bernays what do you think?

Reply:
I think it's a little too detailed for this post, but I get that it helps to manage expectations. If we keep it we may just want to call out that these are limitations during the beta phase, and we intend to make this easier to manage in the future.